avocado-setup.py

```diff
@@ -650,14 +679,17 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
         test_suites = args.run_suite.split(',')
         if args.install_guest:
             test_suites.insert(0, 'guest_install')
+        print ("\n in")
+        configure_ffdc()
```

Comment:
Why not call it immediately after cloning the git repo ?

Reply:
I thought to call this on every time when Test called not on boot strap

Reply:
Is it needed for every test run ?

Reply:
ah sorry, I was not  clear, what I meant as every test call (bucket/s) we can call this

Reply:
@PraveenPenguin I have a doubt here. Will the system ffdc expected to change before every test run ? If yes, why ?

My thought here is: customer are expected to run ffdc once on system setup, and then not worry about it. We should replicate that, and anything more than this should be covered by ffdc test case and not wrapper.

Reply:
thanks for review , sure make sense, as discussed will send V2 soon